### PR TITLE
shrink test_parallel_executor size

### PR DESCRIPTION
**Pull request by panyx0718**

### CODE REVIEW

File: python/paddle/fluid/tests/unittests/test_parallel_executor.py [hunk 1]
```diff
@@ -150,9 +150,9 @@ def SE_ResNeXt152Small(batch_size=2):
     conv = fluid.layers.pool2d(
         input=conv, pool_size=3, pool_stride=2, pool_padding=1, pool_type='max')
 
-    cardinality = 64
+    cardinality = 32
     reduction_ratio = 16
-    depth = [3, 8, 36, 3]
+    depth = [3, 4, 6, 3]
```
Comment (wangkuiyi) [inline python/paddle/fluid/tests/unittests/test_parallel_executor.py]: I vaguely remember this kind of unit test -- checking that a function doesn't fail with a certain number of combinations of parameter values without special meaning (like edge cases) -- doesn't help much and should be avoided? @panyx0718 @chengduoZH

Comment (panyx0718) [reply python/paddle/fluid/tests/unittests/test_parallel_executor.py]: Agree. I think the test author can update it when he has time. Now, I'm just make the test run faster.

**Comment by panyx0718** [comment]
I accidentally sync the branch with upstream. github detects no change and closed the PR.
I opened a new one:
https://github.com/PaddlePaddle/Paddle/pull/9678/files
